internal/kms/test/gcp_test.go

```diff
@@ -0,0 +1,81 @@
+//go:build integration
+
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package test
+
+import (
+	"context"
+	"flag"
+	"testing"
+	"time"
+
+	"github.com/edgelesssys/constellation/v2/internal/kms/kms/gcp"
+	"github.com/edgelesssys/constellation/v2/internal/kms/storage"
+	"github.com/edgelesssys/constellation/v2/internal/kms/uri"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGCPKMS(t *testing.T) {
+	if !*runGcpKms {
+		t.Skip("Skipping Google KMS test")
+	}
+	if *gcpProjectID == "" || *gcpLocation == "" || *gcpKeyRing == "" || *kekID == "" {
+		flag.Usage()
+		t.Fatal("Required flags not set: --gcp-project, --gcp-location, --gcp-keyring, --kek-id")
+	}
+	require := require.New(t)
+
+	store := storage.NewMemMapStorage()
```

Comment:
Why use MemMapStorage here, but cloud storage for AWS/Azure?

Reply:
What do you mean?
AWS and Azure also use MemMapStorage for the KMS tests, don't they?

Reply:
Whoops I was confused by the order of the tests in the different files I guess. Nvm.